common/config/rush/pnpm-lock.yaml

```diff
@@ -4641,6 +4665,14 @@ packages:
       debug: '*'
     resolution:
       integrity: sha512-pCB8eNxGgdIdZeC885rbhZ/VyuOPNHUIDNL9EaaMf1NVzpvTjMO8a7zRTn51ZJhOOOxCSpalUdT1A8x76LyVqg==
+  /keytar/7.0.0:
```

Comment:
This is necessary until we figure out how to move these optional dependencies out of this repo. It would be too hard for me to address this at any extent as part of this PR.